.eslintignore

```diff
@@ -7,8 +7,9 @@ core/
 **/public/**
 **/build/**
 **/fixtures/**
+**/lib/**
```

Comment:
This seems overly broad?

Reply:
How so? We don't want to lint the package build directories. It's not `src` code, it's transpiled build output.

Reply:
This will ignore 

```
core/sgx/bigdecimal-rs/src/lib.rs
core/sgx/enclave/src/lib.rs
core/sgx/libadapters
core/sgx/libadapters/src/lib.rs
core/sgx/utils/src/lib.rs
evm/box/contracts/lib
tools/sa_requester/lib
```

Which may be more broad? Maybe something like `node_modules/**/lib` ?

Reply:
Why are node_modules included in our repository?  Isn't versioning handled by package.json or yarn?

Reply:
@felder-cl These are lint ignore rules which apply to your working directory, not the repository.

Reply:
@j16r the rust files + contracts can be ignored. I'll need to convert `tools/sa_requester/lib` to come from `src`

Reply:
Yeah, I'm just worried that these masks are too broad and will allow code to sneak past the linter very easily. I was kind of hoping that javascript code isn't so dispersed over our project that you'd need a mask like `**/lib/**`. I'd expect to see ignores like `node_modules/`, `explorer/src/fixtures`.

Reply:
Would ` **/lib/**.js` work?

Reply:
@se3000 that would skip `tools/sa_requester/lib/index.js`

Reply:
@rupurt by our new standards, shouldn't that be moved to `tools/sa_requester/src/index.js` instead of `lib` now?

Reply:
@se3000 gotcha. Yes, I think it should be moved to `src`.

Your pattern `**/lib/**.js` would work. `**/lib/**` would also work, this would also work for `.ts` files if they ever get included in build output. `.rs` files will never match in our `eslint` task.

Reply:
#2334